app/app.component.ts

```diff
@@ -1,14 +1,56 @@
 import {Component} from 'angular2/core';
+import {RouteConfig, ROUTER_DIRECTIVES} from 'angular2/router';
+import {Header} from './shared/components/header/header.component';
+import {Footer} from './shared/components/footer/footer.component';
+import {Root} from './sections/root/root.component';
+import {Videos} from './sections/videos/videos.component.ts';
+import {Downloads} from './sections/downloads/components/downloads.component';
+import {ContentfulConfig} from './app.constans';
+import {About} from './sections/about/about.component';
+
 
 @Component({
   selector: 'gapminder-app',
+  directives: [Header, Footer, ...ROUTER_DIRECTIVES],
+  styles: [`
+    #container {
```

Comment:
Maybe it would be better to keep styles in separate files and use `styleUrls` attribute to load them